extensions/amp-analytics/amp-analytics.md

```diff
@@ -27,70 +27,72 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong>Examples</strong></td>
-    <td><a href="https://ampbyexample.com/components/amp-analytics/">Annotated code example for amp-analytics</a></td>
+    <td>See AMP By Example's <a href="https://ampbyexample.com/components/amp-analytics/">amp-analytics example</a>.</td>
   </tr>
 </table>
 
 [TOC]
 
-## Overview
+## Sending analytics to a vendor or in-house?
 
-You can use the `<amp-analytics>` element to measure activity on an AMP document. In the `<amp-analytics>` element, you specify a JSON configuration object that contains the details for what to measure and where to send the analytics data. You can send the tracking data to an analytics vendor and/or to a URL. There are many analytics vendors that are pre-configured for `<amp-analytics>`, see [Analytics Vendors](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html) for details.
+Before you start using AMP analytics on your site, you need to decide whether you will you use third-party analytics tools to analyze user engagement, or your own in-house solution.
 
-**Example**
+{% call callout('Read on', type='read') %}
+Learn all about AMP analytics in the [Configure Analytics](https://www.ampproject.org/docs/guides/analytics_amp) guide.
+{% endcall %}
+
+### Sending data to an analytics vendor <a name="analytics-vendors"></a>
+
+AMP analytics is specifically designed to measure once and report to many. If you are already working with one or more analytics vendors, check the list of [Analytics Vendors](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html) to see if they’ve integrated their solution with AMP.
+
+
+To send data to a supported analytics vendor:
+
+1.  In the `<amp-analytics>` tag, add the `type`attribute and set its value to the specified [vendor](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html).
+2. Determine what data you want to capture and track, and specify those details in the configuration data. See the vendor's documentation for  instructions on how to capture analytics data.
 
-In the following example, we send analytics data to `https://example.com/analytics` when the AMP document is first loaded, and each time an `<a>` tag is clicked:
+
+*Example: Sending data to a third-party analytics provider*
+
+In the following example, pageview data is sent to Google Analytics, a third-party analytics provider.  Details for sending pageview data to Google Analytics can be found in the [Google Analytics](https://developers.google.com/analytics/devguides/collection/amp-analytics/#page_tracking) documentation.
```

Comment:
I think having the example is great, but we shouldn't make Google Analytics special as they are only one of many.